<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShopWidget</class>
 <widget class="QWidget" name="ShopWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="bomb">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>bomb</string>
   </property>
  </widget>
  <widget class="QPushButton" name="horizon">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>horizon</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vertical">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>vertical</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Buy">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Buy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>751</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">


QLabel {
    background-color: #3E2723;  /* 深咖啡色背景 */
    color: white;  /* 白色文字 */
    border: 2px solid #5D4037;  /* 边框颜色为稍浅的咖啡色 */
    border-radius: 5px;  /* 圆角效果 */
    padding: 10px;  /* 内边距 */
    text-align: center;  /* 文字居中 */
    font-size: 14px;  /* 字体大小 */
font: 18pt &quot;Broadway&quot;;
}


QLabel:pressed {
    background-color: #5D4037;  /* 点击时背景色变为更深的咖啡色 */
    border-color: #3E2723;  /* 点击时边框颜色变深 */
}
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;SHOP&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>按esc返回游戏</string>
   </property>
  </widget>
  <widget class="QLabel" name="introduction">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="picture">
   <property name="geometry">
    <rect>
     <x>409</x>
     <y>131</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="score">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bombLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="horizonLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="verticalLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>330</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
